### DISCUSSION TITLE
Merge of virtual and physical battery

### DISCUSSION THREAD

**Discussion by JShomeassistant** [mr-manuel/venus-os_dbus-mqtt-battery#31] [Q&A]
Hi community.
I have a question regarding the view of the Remote Console.
I implemented 1 batterie-pack of two EEL Batteries in VRM. That works very fine. Now I added a second batterie via this addon as a virtual batterie. I get every information via MQTT to VRM and I see every information in the Remote Console.
But now the both batteries are merged and I just see one "big" batterie. 

Is there an option to divide the batteries and see 2 icons in Remote Console?

BR
Jochen

**Comment by mr-manuel** [comment]
No
